TanzuForVS/TanzuToolkitForVisualStudioPackage.cs

```diff
@@ -116,7 +119,7 @@ private void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IViewLocatorService, WpfViewLocatorService>();
             services.AddSingleton<IDialogService, WpfDialogService>();
             services.AddSingleton<ICfCliService, CfCliService>();
-            services.AddSingleton<IFileLocatorService, FileLocatorService>();
+            services.AddSingleton<IFileLocatorService>(new FileLocatorService(assemblyBasePath));
```

Comment:
Could you keep original code here and move assembly lookup/resolution to FileLocalService?

Reply:
Yeah I think technically we could... This change was initially meant to facilitate injecting mock values during unit testing for the FileLocatorService -- I'm sure there are other ways to accomplish that (e.g. an extra ctor which takes this param & overwrites that value for testing purposes), but I kinda like the idea that the VSIX package is the one to tell this service where the VSIX assemblies live. I'm not super passionate about either strategy though; would you say one makes a lot more sense than the other?

Reply:
no real preference, more curious to your thought behind making change where you did.    I think both reasons for either impl are valid.